app.rb

```diff
@@ -1,19 +1,81 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
 apartments = data[:apartments]
 tenants = data[:tenants]
+ruby_apartments = []
+ruby_tenants = []
+
+ruby_apartments = apartments.each do |id, address, rent, size|
+   ruby_apartments.push(Apartment.new(id, address, rent, size))
+end
```

Comment:
This works I think, but you've sort of mixed up `map` and each here. I'd say the ideal use looks like either of these:

```ruby
# Using map to create a new array
ruby_apartments = apartments.map do |unit|
  Apartment.new(unit[:id], unit[:address], unit[:monthly_rent], unit[:square_feet])
end

# OR using each + pusing to the array

apartments.each do |unit|
  unit_obj = Apartment.new(unit[:id], unit[:address], unit[:monthly_rent], unit[:square_feet])
  ruby_apartments << unit_obj
end
```